packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -21,10 +26,41 @@ import DisplayId from '../util/DisplayId';
 
 import { StatusBadge } from './StatusBadge';
 
-export let ROW_HEIGHT = 43;
+type SchedulesTableProps = {
+  schedules: ScheduleEntity[];
+  statuses: ScheduleStatuses;
+  filter: string;
+  allowCompleted: boolean;
+  onSelect: (id: ScheduleEntity['id']) => void;
+  onAction: (actionName: ScheduleItemAction, id: ScheduleEntity['id']) => void;
+  style: CSSProperties;
+  minimal?: boolean;
+  tableStyle?: CSSProperties;
+};
 
-function OverflowMenu({ schedule, status, onAction }) {
-  let [open, setOpen] = useState(false);
+type CompletedScheduleItem = { id: 'show-completed' };
+type SchedulesTableItem = ScheduleEntity | CompletedScheduleItem;
+
+export enum ScheduleItemAction {
+  PostTransaction = 'post-transaction',
+  SkipSchedule = 'skip',
+  CompleteSchedule = 'complete',
+  RestartSchedule = 'restart',
+  DeleteSchedule = 'delete',
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we generally use union types everywhere and not enums. Could you please move this back?